HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -35,7 +35,7 @@ def test_hvac_configurations
     args_hash = { 'hpxml_path' => File.absolute_path(@tmp_hpxml_path),
                   'skip_validation' => true }
     Dir["#{@sample_files_path}/base-hvac*.xml"].each do |hvac_hpxml|
-      next if hvac_hpxml.include? 'autosize'
+      next if (hvac_hpxml.include? 'autosize') || (hvac_hpxml.include? 'autosizing-factor')
```

Comment:
How about renaming the HPXML from 'autosizing-factor' to 'autosize-factor'? Then we don't need this.